docs/userguide.md

```diff
@@ -1,205 +1,206 @@
 # TiSpark (version >= 2.0) User Guide
 
-**Note: This is a user guide for TiSpark version >= 2.0. If you are using version before 2.0, please refer to [Document for Spark 2.1](./userguide_spark2.1.md)**
-
-TiSpark is a thin layer built for running Apache Spark on top of TiDB/TiKV to answer the complex OLAP queries. It takes advantages of both the Spark platform and the distributed TiKV cluster, at the same time, seamlessly glues to TiDB, the distributed OLTP database, to provide a Hybrid Transactional/Analytical Processing (HTAP) solution to serve as a one-stop solution for online transactions and analysis.
-
-
-TiSpark depends on the TiKV cluster and the PD cluster. It also needs to set up a Spark cluster. This document provides a brief introduction to how to set up and use TiSpark. It requires some basic knowledge of Apache Spark. For more information, please refer to [Spark website](https://spark.apache.org/docs/latest/index.html).
+> **Note:**
+>
+> This is a user guide for TiSpark version >= 2.0. If you are using version earlier than 2.0, refer to [Document for Spark 2.1](./userguide_spark2.1.md)
 
+This document introduces how to set up and use TiSpark, which requires some basic knowledge of Apache Spark. Refer to [Spark website](https://spark.apache.org/docs/latest/index.html) for details.
 
 ## Overview
 
-TiSpark is an OLAP solution that runs Spark SQL directly on TiKV, the distributed storage engine.
+TiSpark is a thin layer built for running Apache Spark on top of TiDB/TiKV to answer the complex OLAP queries. It enjoys the merits of both the Spark platform and the distributed clusters of TiKV, at the same time, seamlessly integrated to TiDB, the distributed OLTP database, to provide one-stop Hybrid Transactional/Analytical Processing (HTAP) solutions for online transactions and analyses.
```

Comment:
Such a long composite sentence...

Some suggestions:
- `integrated -> integrates`? If you mean by `it enjoys the merits while integrating to tidb`.
- `to provide -> provides`? If you mean by `it enjoys the merits while providing solutions`.
- Or change `at the same time` to `via/by`? If the following is considered an attributive clause.
- Or we may simply split this sentence into two for better understanding...

Reply:
What about `While enjoying the merits of both the Spark platform and the distributed clusters of TiKV, it is seamlessly integrated into TiDB, the distributed OLTP database, and provide one-stop Hybrid Transactional/Analytical Processing (HTAP) solutions for online transactions and analyses.`?

Reply:
Great! Just replace `it` with `TiSpark`.

Reply:
I'm just wondering that `providing one-stop solutions` may be a result of `integrated into TiDB`, so that they should not conjunct as parallel.

Reply:
Also, I believe `integrate with` is better than `integrate into`. @TomShawn

Reply:
Yes, you are right. What about `While enjoying the merits of both the Spark platform and the distributed clusters of TiKV, it is seamlessly integrated with TiDB, the distributed OLTP database, and thus blessed to provide one-stop Hybrid Transactional/Analytical Processing (HTAP) solutions for online transactions and analyses.`?

Reply:
nice